extensions/mssql/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "4.0.0.11",
+	"version": "4.1.0.1",
```

Comment:
should this be [4.1.0.2](https://github.com/microsoft/sqltoolsservice/releases/tag/4.1.0.2)? 4.1.01 is from main

Reply:
4.1.0.2 was created by mistake. I've deleted it.
4.1.0.1 is from main and I want it to be in ADS main. for ADS release branch, I will create a PR to update it with the STS version created from the STS release/4.0 branch.